consensus/hotstuff/verification/staking_signer.go

```diff
@@ -23,16 +23,14 @@ type StakingSigner struct {
 
 // NewStakingSigner creates a new combined signer with the given dependencies:
 // - the staking signer is used to create and verify aggregatable signatures for Hotstuff
-// - the signer ID is used as the identity when creating signatures;
 func NewStakingSigner(
 	staking module.Local,
```

Comment:
minor: we could call this field `me` like we do for other `module.Local` fields in other files.

Reply:
+1 on [Tarak's comment](https://github.com/onflow/flow-go/pull/1706#discussion_r763409179)